hca/dss/util/__init__.py

```diff
@@ -1,11 +1,8 @@
 import errno
-import os
-
 import platform
+import os
```

Comment:
Nit: Moving `import os` here complicates the diff and breaks the alphabetical ordering (which PyCharm inserts by default)